### PR TITLE
Fix flushing when app crashed or exited

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

**Comment by samdark** [comment]
https://github.com/yiisoft/yii-debug/pull/171 fixes it in a better way.

File: src/Debugger.php [hunk 1]
```diff
@@ -49,8 +49,6 @@ public function startup(object $event): void
             $this->target->addCollector($collector);
             $collector->startup();
         }
-
-        register_shutdown_function([$this, 'shutdown']);
```
Comment (vjik) [inline src/Debugger.php:53]: It's need keep, because run shutdown logic.

File: src/Storage/FileStorage.php [hunk 2]
```diff
@@ -34,6 +34,7 @@ final class FileStorage implements StorageInterface
 
     public function __construct(private string $path, private FilesystemInterface $filesystem, private DebuggerIdGenerator $idGenerator, private Aliases $aliases, private array $excludedClasses = [])
     {
+        register_shutdown_function([$this, 'flush']);
```
Comment (vjik) [inline src/Storage/FileStorage.php:37]:
```suggestion
```

File: src/Debugger.php (hunk 1 above)
Comment (rustamwin) [reply src/Debugger.php:53]: It's doesn't makes sense except storage flushing.

**Pull request closed without merging**